server/app/import.go

```diff
@@ -20,8 +20,9 @@ import (
 )
 
 const (
-	archiveVersion  = 2
-	legacyFileBegin = "{\"version\":1"
+	archiveVersion    = 2
+	legacyFileBegin   = "{\"version\":1"
+	importMaxFileSize = 1024 * 1024 * 70
```

Comment:
Where is this file size coming from? Should this be from the configuration setting?

Reply:
This comes directly from the Mattermost [slack import service](https://github.com/mattermost/mattermost/blob/master/server/platform/services/slackimport/slackimport.go#L78), but I'm happy to make it configurable

Reply:
After taking a look at how the configuration is generated, making this easy to configure from the plugin's perspective is a bit more involved than expected. We've agreed on reusing MM's approach and we can revisit and expand it later if necessary